src/word_debt_bot/game/core.py

```diff
@@ -1,50 +1,85 @@
 import json
+import math
 import os.path
 import pathlib
 from dataclasses import asdict
+from datetime import datetime, timedelta
+
+from word_debt_bot.game.state import WordDebtState
 
 from .player import WordDebtPlayer
 
+CURRENT_SERIALIZATION_VERSION = 1
+
+
+def do_state_migration(state: dict):
+    if not state.get("version"):  # Implicit version 0: Users are entire state
+        return {
+            "version": 1,
+            "users": state,
+            "modifiers": [],
+        }
+    if state["version"] != 1:
+        raise ValueError("Unknown state version -- Unable to initialize")
+    return state
+
 
 class WordDebtGame:
     def __init__(self, state_file_path: pathlib.Path):
         self.path = state_file_path
         self.init_state()
+        self._prune_expired_modifiers()
```

Comment:
Init only happens once per bot run, need to figure out another time point to trigger pruning (though every command run is overkill).